pkgs/by-name/ru/rustdesk-flutter/package.nix

```diff
@@ -163,7 +182,19 @@ flutter319.buildFlutterApplication rec {
     ".."
   ];
 
-  patches = [ ./make-build-reproducible.patch ];
+  patches = [
+    ./make-build-reproducible.patch
+  ];
+
+  prepareBuildRunner = ''
+    cp ${./build-runner.sh} build_runner
+    substituteInPlace build_runner \
+      --replace-fail "@bash@" "${lib.getExe bash}"
```

Comment:
This is not how it should be done, you should get the runtime shell from the stdenv